README.md

```diff
@@ -29,284 +30,57 @@ pip install graphdatascience
 ```
 
```

Comment:
I wonder if we should also have a `## Documentation` header under which we refer to the manual too, just to be abundantly clear?

Reply:
Good idea!